src/tests/Regressions/coreclr/GitHub_67144/test67144.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This pattern is used for regression tests that should prevent the issue from coming back. I find it useful to be able to map the test name to the issue number directly in case the test fails. I can then quickly see if the fix for the issue was incomplete or the problem is somehow different. 
But if you'd still prefer me to move it to the location you've suggested, please let me know and I'll do that.

Reply:
I would still prefer it moved/renamed. I can see the use case of the easy mapping based on test name, but my personal preference is having the name indicate what it is testing (and then being able to see relevant issue number(s) in comments if desired).